source/reference/minio-mc-admin/mc-admin-prometheus-generate.rst

```diff
@@ -90,18 +129,97 @@ Global Flags
    :end-before: end-minio-mc-globals
 
 
-Example
--------
+Examples
+--------
+
+Generate a default metrics v3 config
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use :mc-cmd:`mc admin prometheus generate --api-version v3` to generate a scrape configuration that collects all v3 metrics for a MinIO deployment:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin prometheus generate ALIAS --api-version v3
+
+- Replace ``ALIAS`` with the :mc-cmd:`alias <mc alias>` of the MinIO deployment.
+
+The output resembles the following:
+
+.. code-block:: shell
+
+   scrape_configs:
+   - job_name: minio-job
+     bearer_token: [auth token]
+     metrics_path: /minio/metrics/v3
+     scheme: http
+     static_configs:
+     - targets: ['localhost:9000']
+
+
+Generate a v3 cluster metrics config
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use :mc-cmd:`mc admin prometheus generate --api-version v3` to generate a scrape configuration that collects v3 cluster metrics for a MinIO deployment:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin prometheus generate ALIAS cluster --api-version v3
+
+- Replace ``ALIAS`` with the :mc-cmd:`alias <mc alias>` of the MinIO deployment.
+
+The output resembles the following:
+
+.. code-block:: shell
+
+   scrape_configs:
+   - job_name: minio-job-cluster
+     bearer_token: [auth token]
+     metrics_path: /minio/metrics/v3/cluster
+     scheme: http
+     static_configs:
+     - targets: ['localhost:9000']
+
+To generate a configuration for a :mc-cmd:`different metric type <mc admin prometheus generate type>`, replace ``cluster`` with the desired type.
+
+
+Generate a v3 bucket replication metrics config
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example generates a scrape configuration for v3 replication metrics of bucket ``mybucket``:
```

Comment:
```suggestion
The following example generates a scrape configuration for v3 replication type metrics of bucket ``mybucket``:
```

I was confused by cluster metrics earlier. I don't think the `type` is as needed here, but if you agree to put it with the cluster metrics, then have to be consistent.